.travis.yml

```diff
@@ -17,6 +17,8 @@ matrix:
     - php: 5.5
       env: SYMFONY_VERSION=2.4.*
```

Comment:
I suggest removing ``2.4.*`` given that it is not maintained anymore (no need to add an extra job for each build for an unmaintained version)